examples/mnist-classification/src/mnist_classification/main.clj

```diff
@@ -14,7 +14,8 @@
 
 (defn -main
   [& args]
+  (println "Welcome! Please wait while we compile some Clojure code...")
   (let [argmap (parse-opts args cli-options)]
-    (require 'suite-classification.core)
-    ((resolve 'suite-classification.core/train-forever-uberjar)
+    (require 'mnist-classification.core)
```

Comment:
It makes the build/run loop *much* faster for uberjar development while adding virtually no additional time during actual running.  I have honestly started with with all uberjar builds because aot isn't worth the time.